File: Assign-Cookies.py
class Solution(object):
    def findContentChildren(self, g, s):
        g.sort()
        s.sort()
        i = 0
        j = 0
        count = 0
        while i < len(g) and j < len(s):
            if s[j] >= g[i]:
                count += 1
                i += 1
                j += 1
            else:
                j += 1
        return count

File: Assign-Cookies.js
/**
 * @param {number[]} g
 * @param {number[]} s
 * @return {number}
 */

var findContentChildren = function(g, s)
 {
    g.sort((a, b) => a - b);
    s.sort((a, b) => a - b);
    let i = 0, j = 0, count = 0;
    while (i < g.length && j < s.length) 
    {
        if (s[j] >= g[i]) 
        {
            count++;
            i++;
            j++;
        } 
        else 
        {
            j++;
        }
    }
    return count;
};
